CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 
 ## [Unreleased]
 
+## [1.8.0] - 2024-10-07
```

Comment:
the admin auth stuff didn't make it into 1.8.0, so that needs to move back to "unreleased"